phoenix-core/src/main/java/org/apache/phoenix/util/IndexUtil.java

```diff
@@ -958,6 +958,13 @@ public static void setScanAttributesForIndexReadRepair(Scan scan, PTable table,
             // This index table must be being deleted. No need to set the scan attributes
             return;
         }
+        // MetaDataClient modifies the index table name for view indexes if the parent view of an index has a child
+        // view. This, we need to recreate a PTable object with the correct table name for the rest of this code to work
+        if (table.getName().getString().contains(QueryConstants.CHILD_VIEW_INDEX_NAME_SEPARATOR)) {
```

Comment:
It would be clearer if you used indexTable rather than table (to be consistent with the logic earlier in the method)

Reply:
Done